NEWS.md

```diff
@@ -1,4 +1,17 @@
 ## 12.9.0 - Unreleased
+### Stories
+
+### Bug Fixes
+
+* [MODORDERS-1090](https://folio-org.atlassian.net/browse/MODORDERS-1090) - Quantity = 0 for electronic format orders
```

Comment:
mentioned twice in News

Reply:
Yes it was mistakenly added in scope of MODORDERS-1090. I supposed that it will be removed during v12.9 release

Reply:
we can remove now